### DISCUSSION TITLE
IEC Wind

### DISCUSSION THREAD

**Discussion by kpramng** [OpenFAST/openfast#1200] [Q&A]
Hi,
I am new to OpenFast and have started using it few weeks back and is still exploring the tool.

The two queries I have are:

1. I want to know how I can run IECWind and where I can find the input files to run IECWind.. Couldn't find the input file here

 https://www.nrel.gov/wind/nwtc/iecwind.html

Can you also help me understand the difference b/w IEC Wind and Inflow Wind.

2. Is there a refrerence or tutorial which I can refer to see how I can run a steady power curve simulation

Best,
Karthik

**Comment by jjonkman** [comment]
Dear @kpramng,

Here are my answers:

1. The file _IECWind_v5.01.02.exe_ at the link you shared is a self-extracting archive containing the compiled binary executable for Windows, the documentation, source code, and sample input file.  IECWind is a simple tool for generating deterministic wind gust event time series defined within the IEC design standards (ECD, EDC, EOG, etc.) in uniform wind file format, which can be used in the InflowWind module of OpenFAST by selecting `WindType` = 2.

2. Are you trying to generate a steady power curve for a rigid wind turbine or a flexible wind turbine (including structural flexibilities)?  The latter would require OpenFAST, e.g., as discussed in the following topic on our forum: https://forums.nrel.gov/t/power-curve/1953.  For a rigid wind turbine you could use the standalone AeroDyn driver.

Best regards,

**Comment by kpramng** [comment]
Hi Jason,

Thanks a lot for the reply.

1. I extracted the files and tried running it via the ommand prompt, but got the below error.

_Program 'IECWind.exe' failed to run: The specified executable is not a valid application for this OS platform.At
line:1 char:1
+ .\IECWind.exe .\IEC.ipt
+ ~~~~~~~~~~~~~~~~~~~~~~~.
At line:1 char:1
+ .\IECWind.exe .\IEC.ipt
+ ~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ResourceUnavailable: (:) [], ApplicationFailedException
    + FullyQualifiedErrorId : NativeCommandFailed_


2- For the steady power curve, I am trying to create it for a flexible wind turbine. I would like to use the 5MW NREL Baseline.

Best regards,
Karthik

**Reply by jjonkman** [reply]
Hi @kpramng,

Other users of IECWind have reported similar problems as discussed in our forum: https://forums.nrel.gov/t/iecwind/198.  Recompiling IECWind from the provided source code should solve the problem.  I recompiled IECWind for Windows and shared a link in my post dated Jan 3, 2022 in this forum topic.

Best regards,

**Reply by kpramng** [reply]
Thanks @jjonkman This helped.
